### PR TITLE
[Kubernetes] Fix Pod dashboard, remove 2 visualisation to migrate to TSDB

### PR DESCRIPTION
**Pull request by tetianakravchenko**
## What does this PR do?

Pod: use last_value instead of average. For counter average is not supported
Api server: 
- remove 2 visualisation, because now when the aggregation was changed sum -> max we need to break it down by multiple fields, but the current limit is 4. Also it is indeed is quite a lot data per one visualisation, we need to rethink current visualisation

TODO:
- [ ] create an issue to readd removed visualisation

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

## Screenshots
This includes both non-TSDB and TSDB data
Screenshot 2023-05-11 at 20 15 51

Screenshot 2023-05-11 at 20 15 04

### CODE REVIEW

File: packages/kubernetes/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.38.0"
```
Comment (tetianakravchenko) [inline packages/kubernetes/changelog.yml:2]: ideally should be merged after https://github.com/elastic/integrations/pull/5992

**Comment by ruflin** [comment]
> we need to break it down by multiple fields, but the current limit is 4.

Is this a Lens specific limitation? Is this something that should get removed?

**Review by MichaelKatsoulis — APPROVED** [review]

**Comment by tetianakravchenko** [comment]
> > we need to break it down by multiple fields, but the current limit is 4.
> Is this a Lens specific limitation? Is this something that should get removed?

yes, seems to be Lens specific, there is max 4 fields for the `Breakdown`, then `Add field` becomes inactive:
Screenshot 2023-05-15 at 09 15 27

I would create a different issue and investigate how/if possible to reorganize those visualisation

**Comment by ruflin** [comment]
@tetianakravchenko Can you file a Kibana issue on this one if this needs changing in the future? https://github.com/elastic/integrations/pull/6171#issuecomment-1547318210

**Pull request merged**